test/integration/app-document/test/csp.js

```diff
@@ -5,7 +5,7 @@ export default (context, render) => {
   describe('With CSP enabled', () => {
     it('should load inline script by hash', async () => {
       const browser = await webdriver(context.appPort, '/?withCSP=hash')
-      if (browser.log) {
+      if (global.browserName !== 'chrome') {
```

Comment:
Shouldn't this be if the browser is chrome 🤔

Reply:
Yeah 😅 did a global replace, updated